ci/manager.sh

```diff
@@ -174,18 +179,30 @@ do
 done
 
 print_header "Obtaining Performance Results from all workers"
-
-rm -f results_summary.stats
-
 for worker_tuple in "${WORKER_LIST[@]}"
 do
     tuple_arr=($worker_tuple)
     worker_ip="${tuple_arr[0]}"
     worker_key_file="${tuple_arr[1]}"
-    scp -i $worker_key_file -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null $worker_ip:stats ./$worker_ip.stats
-    check_exit_code "ERROR: Failed to fetch results from $worker_ip"
-    # TODO: this will overwrite results if we have more  than 1 worker, investigate this case
-    python3 speed-tester-analysis.py ./$worker_ip.stats $worker_ip results_summary.stats
+    # get the benchmarks for each node (some servers are faster)
+    . ./$worker_ip/benchmarks
```

Comment:
what does this do btw? Can't find a reference to benchmarks

Reply:
This is not included (on purpose) in the Github code. For each node, there is a folder that is run, storing their specific data, because not every set of nodes will have the same generic performance of CI. This allows us to benchmark based on a different set of averages. I included this info in the README.md, for future development.

Reply:
Hm where exactly is that info?

Reply:
Where `$worker_ip` is in the config file (nimbnode17 for example), the folder is in `/openNetVM/ci/nimbnode17/` so it's just a text file in there with server-specific statistics.